<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLChOscpWidget</class>
 <widget class="QWidget" name="WLChOscpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Channel</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="cbElement"/>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="sbIndex"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbTypeData"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="color">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
